built-in-policies/README.md

```diff
@@ -69,4 +69,4 @@ Here are the changes that occur when a policy gets out of preview:
 For customers who do not want such changes to impact their policy definitions, we recommend duplicating built-in policy definitions & assigning them as custom definitions. Customers can also choose to be notified of updates by [subscribing](https://docs.github.com/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#configuring-your-watch-settings-for-an-individual-repository) to this Github repository.  
  
 ## Contributing
-Changes can not be made to built-in policies directly in this repo. If you find an issue in a built-in policy, feel free to open a PR with the proposed fix, [open an issue](https://github.com/Azure/azure-policy/issues/new/choose), or [open a Microsoft Azure support ticket](https://azure.microsoft.com/support/create-ticket/). The change will be made out-of-band and will be represented in this repo after the next built-in policy release.
+Changes can not be made to built-in policies directly in this repo. If you find an issue in a built-in policy, feel free to [open an issue](https://github.com/Azure/azure-policy/issues/new/choose), or [open a Microsoft Azure support ticket](https://azure.microsoft.com/support/create-ticket/). The change will be made out-of-band and will be represented in this repo after the next built-in policy release.
```

Comment:
Recommend slightly different wording: "The change will be made out-of-band..." => "Changes to built-ins are made out-of-band...".